mocks/beehive/fake_module.go

```diff
@@ -0,0 +1,78 @@
+// Code generated by MockGen. DO NOT EDIT.
```

Comment:
I am unable to find any place where reference this package? I can't find any reason to generate this mock package, am I missing something?

Reply:
Please see msg_processor_test.go Line31 where we have imported this package and used for creating fake modules.

Reply:
Okay, it's useful :)